mixitup/teamInfo.js

```diff
@@ -0,0 +1,247 @@
+var context = 
```

Comment:
Is this method okay? You didn't say anything which leads me to believe that you understand my comment above and don't have any problems with it.